oofcanvas 1.0.2

snarkhunter opened the pull request:
#### Description

<!-- Note: it is best to make pull requests from a branch rather than from master -->

###### Type(s)
<!-- update (title contains ": U(u)pdate to"), submission (new Portfile) and CVE Identifiers are auto-detected, replace [ ] with [x] to select -->

- [x] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on
<!-- Triple-click and copy the next line and paste it into your shell. It will copy your OS and Xcode version to the clipboard. Paste it here replacing this section.
sh -c 'printf "%s\n" "macOS `sw_vers -productVersion` `sw_vers -buildVersion` `uname -m`" "`xcodebuild -version|awk '\''NR==1{x=$0}END{print x" "$NF}'\''`"'|tee /dev/tty|pbcopy
-->
macOS 11.6.2 20G314 x86_64
Xcode 13.2 13C90

###### Verification <!-- (delete not applicable items) -->
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? <!-- Please don't open a new Trac ticket if you are submitting a pull request. -->
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

<!-- Use "skip notification" (surrounded with []) to avoid notifying maintainers -->

snarkhunter commented:
I don't know why it says I've asked to merge 17 commits.  I made a new branch, which was up to date with master, and then made one commit on that branch.  What am I missing?

pmetzger commented:
> I don't know why it says I've asked to merge 17 commits.

18 actually. Have a look at them. I'd start over.

snarkhunter commented:
I apparently don't know enough about git.  I thought I had started over!  I updated my fork on github and the clone of that fork on my desktop.  I created a new branch and added the new Portfile on that branch on the desktop, and pushed that branch to github.  When I did that it gave me a link  for creating a pull request, which I followed.  What am I missing?

pmetzger commented:
I don't know what you're missing, but clearly you did something wrong (and twice).

FWIW, I highly recommend the online book [Pro Git](https://git-scm.com/book/en/v2); it's a great resource.

snarkhunter commented:
Thanks, I'll look at the book.

On 1/31/22, 1:58 PM, "Perry E. Metzger" ***@***.******@***.***>> wrote:


I don't know what you're missing, but clearly you did something wrong (and twice).

FWIW, I highly recommend the online book Pro Git<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgit-scm.com%2Fbook%2Fen%2Fv2&data=04%7C01%7Cstephen.langer%40nist.gov%7C3b7ea5b69a0340b4fcef08d9e4ebacec%7C2ab5d82fd8fa4797a93e054655c61dec%7C1%7C0%7C637792523148157770%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=0OS0Xs1IR7Vi5srvyhn4vnMmp7sDgCP2D9KOA83e7bk%3D&reserved=0>; it's a great resource.

—
Reply to this email directly, view it on GitHub<https://github.com/macports/macports-ports/pull/13802#issuecomment-1026103672>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABZFHR2MG7WW6VAYVUDQMZ3UY3LVLANCNFSM5NBXFNIA>.
Triage notifications on the go with GitHub Mobile for iOS<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cstephen.langer%40nist.gov%7C3b7ea5b69a0340b4fcef08d9e4ebacec%7C2ab5d82fd8fa4797a93e054655c61dec%7C1%7C0%7C637792523148157770%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=jDGVvt2fnORxFmiMT8c5seQJDw8lqiWVmTNAhoRs6qk%3D&reserved=0> or Android<https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cstephen.langer%40nist.gov%7C3b7ea5b69a0340b4fcef08d9e4ebacec%7C2ab5d82fd8fa4797a93e054655c61dec%7C1%7C0%7C637792523148157770%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=kWAD859XOGteh4gzY9xoaY8YPjgYntsUUZRRzQmLrA0%3D&reserved=0>.
You are receiving this because you authored the thread.Message ID: ***@***.***>
